roles/baseline/defaults/main.yml

```diff
@@ -90,6 +90,26 @@ NFS_CLIENT_CONFIG:
     archiveOnDelete: "false"
     name: sas
 
+## pg-storage storage class config
+PG_NFS_CLIENT_NAME: nfs-subdir-external-provisioner-pg-storage
```

Comment:
Need to update the names in `CONFIG-VARS.md` as well

Reply:
Nevermind didn't realize this were new variables, and the old ones still exist. Do we need to doc these?

Reply:
I will add doc to include the new PG_ set of variables in `CONFIG-VARS.md` and update the default for NFS_CLIENT_NAME as well.